docs/how-to/deploy_to_different_channels.md

```diff
@@ -0,0 +1,56 @@
+# Deploy your bot to different platforms
+
+To link a channel to your bot:
+
+!!! info "Note"
+    Not all channels require a provider.
+
+- Choose a [provider][2] for your channel.
+- [Configure a messaging provider][6]. You will need to get the required credentials from your chosen provider.
+- Once the provider is set up, navigate to your bot on OCS and click the plus icon in the "Channels" section.
+- Choose your channel and complete the form. Follow the guide below to get the required information for each channel.
+
+## Web and API
+The web channel uses the web interface and is enabled by default for all bots.
```

Comment:
```suggestion
The web channel uses the web interface and is enabled by default for all bots. Likewise, all bots can be accessed via the APIs.
```

(Maybe add a link to a placeholder page for APIs?)

Reply:
Added an API page as well.
https://github.com/dimagi/open-chat-studio-docs/pull/20/commits/adf36a9cdc0934dac74496b1591006aeecbe05dc